docs/articles/expensify-classic/connections/certinia/Certinia-Troubleshooting.md

```diff
@@ -1,6 +1,79 @@
+
+# ExpensiError FF0061: Object validation has failed. The credit terms…
+To resolve the error "Object validation has failed. The credit terms for the selected account on this document are not correctly defined," follow these steps:
+
+1. Identify the account used for the report being exported. This could be the account on the project for PSA/SRP or the account linked to the resource for FFA.
+
+2. In Certinia, under this account:
+   - Ensure that Base Date 1 is configured to invoice.
+   - Set Days Offset to 1 or more.
+   - Verify that a currency is selected for the account.
+
+By following these guidelines, you can correct the issue and ensure proper validation of the credit terms for the selected account.
+
+# ExpensiError FF0074: You do not have permissions for this resource
+This error message indicates a requirement to establish permission controls for the report creator/submitter within Certinia.
+
+To set this up:
+1. Navigate to Permission Controls in Certinia.
+2. Select "New" to create a new permission control.
+3. Enter the User and Resource Fields.
+4. Ensure all necessary permission fields are checked or configured appropriately.
+
+By completing these steps, you can effectively manage and grant the required permissions to the report creator/submitter in Certinia.
+
+# ExpensiError FF0076: Could not find employee in Certinia
+Go to Contacts in Certinia and add the report creator/submitter's Expensify email address to their employee record, or create a record with that email listed.
+
+If a record already exists then search for their email address to confirm it is not associated with multiple records.
+
+# ExpensiError FF0089: Expense Reports for this Project require an Assignment
+This error indicates that the project needs to have the permissions adjusted in Certinia
+
+Go to Projects > [project name] > Project Attributes and check Allow Expense Without Assignment. 
+
+# ExpensiError FF0091: Bad Field Name — [field] is invalid for [object]
+This means the field in question is not accessible to the user profile in Certinia for the user whose credentials were used to make the connection within Expensify. 
+
+To correct this:
+* Go to Setup > Build > expand Create > Object within Certinia
+* Then go to Payable Invoice > Custom Fields and Relationships
+* Click View Field Accessibility 
+* Find the employee profile in the list and select Hidden
+* Make sure both checkboxes for Visible are selected
+
+Once this step has been completed, sync the connection within Expensify by going to **Settings** > **Workspaces** > **Groups** > _[Workspace Name]_ > **Connections** > **Sync Now** and then attempt to export the report again. 
+
+# ExpensiError FF0132: Insufficient access. Make sure you are connecting to Certinia with a user that has the 'Modify All Data' permission
+
+Log into Certinia go to Setup > Manage Users > Users and find the user whose credentials made the connection. 
```

Comment:
Grammar seems off here? specifically "Log into Certinia go to ..." - maybe should be a line break here?

Reply:
I think a comma would probably suffice or an "and". This doc is a copy paste with the missing elements of vanilla added to get it migrated asap before Vanilla is shut down. It has not had a full rewrite yet, some of these docs are a dogs breakfast, but we have a hard deadline to migrate. Feel free to correct it if you think it's off.